app/src/pages/datasets/DatasetActionMenu.tsx

```diff
@@ -56,13 +56,6 @@ export function DatasetActionMenu(props: DatasetActionMenuProps) {
         onError: (error) => {
           onDatasetDeleteError(error);
         },
-        updater: (store) => {
```

Comment:
figured it out. Need to pass a fetch policy to refetch

Reply:
Nice. Will this approach fix the manual refetches we do on project deletion, too?

Reply:
I don't think so